src/commands/rage.ts

```diff
@@ -0,0 +1,63 @@
+import supportsColor from 'supports-color';
```

Comment:
mind using the named export from chalk instead so we don't have to bring in a separate dep?

([source](https://github.com/chalk/chalk?tab=readme-ov-file#supportscolor))